app/controllers/foo_controller.rb

```diff
@@ -1,6 +1,8 @@
 class FooController < ActionController::Base  
   def new_user
     user = User.first(:conditions => ("username = '#{params[:username]}'"))
+    user = User.first(:conditions => ("username = '#{params[:username]}'"))
```

Comment:
__[staging][rails_best_practices]__
* move finder to named_scope [:ref](http://rails-bestpractices.com/posts/2010/07/14/move-finder-to-named_scope/)

<!-- Comment by [SideCI](https://sideci.com) -->